api/src/main/java/io/minio/errors/BucketPolicyTooLargeException.java

```diff
@@ -16,23 +16,10 @@
 
 package io.minio.errors;
 
-/** Thrown to indicate that given bucket name is not valid. */
+/** Thrown to indicate that received bucket policy is larger than 12KiB size. */
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html

according to AWS S3 and MinIO the max size is 20 KiB

Reply:
Looks like it is increased from 12KB to 20KB